test/scenarios/RoughLiquidation.spec.ts

```diff
@@ -0,0 +1,107 @@
+import { parseEther, parseUnits } from 'ethers/lib/utils';
```

Comment:
Extra logs, no asserts.
Can also be removed, negative liquidation covered in scenario.